cypress/e2e/movies.cy.ts

```diff
@@ -0,0 +1,10 @@
+describe('template spec', () => {
+  it('passes', () => {
+    cy.visit('http://localhost:3000/');
+    cy.get('input').type('gold');
+    cy.contains('Search').click();
+
+    cy.url().should('include', '/search/gold');
+    cy.get('div[class^="movie-items_movies"]').should('contain', 'Golden');
```

Comment:
Hm-hm, XPath could be used, however, if you decide to change your layout, your test might get failed. I would advise to just use `.movie-items_movies` selector, it should work